lib/prima_auth0_ex/token_provider/encrypted_redis_token_cache.ex

```diff
@@ -77,7 +83,14 @@ defmodule PrimaAuth0Ex.TokenProvider.EncryptedRedisTokenCache do
 
   defp build_token(_), do: {:error, :malformed_cached_data}
 
-  defp enabled?, do: :prima_auth0_ex |> Application.get_env(:client, []) |> Keyword.get(:cache_enabled, true)
-  defp namespace, do: Application.fetch_env!(:prima_auth0_ex, :client)[:cache_namespace]
+  defp cache_enabled?,
+    do: Config.redis(:cache_enabled, true)
+
+  defp namespace(:default_client),
+    do: Config.default_client!(:cache_namespace)
```

Comment:
Can't we derive this from Atom.to_string?

Reply:
I was thinking of doing something like `"#{client |> Atom.to_string() |> String.replace("_", "-")}-namespace"`, but at the same time I like having the namespace be user-provided and I'm starting to think it's required

If two services use the `:default_client` they would end up trying to save everything under the same namespace which is not ideal